app/src/main/java/org/fossasia/susi/ai/rest/responses/others/WebIcon.kt

```diff
@@ -1,19 +1,15 @@
 package org.fossasia.susi.ai.rest.responses.others
 
-import com.google.gson.annotations.Expose
-import com.google.gson.annotations.SerializedName
-
 /**
  * <h1>Kotlin Data class to parse web icon object in retrofit response from websearch client.</h1>
  *
  * Created by mayank on 12-12-2016.
  */
 
-class WebIcon {
+class WebIcon (
+
 
-@SerializedName("URL")
- @Expose
  var url: String? = null
```

Comment:
@S2606 this is a single variable in whole file. Can we like add three related classes like webIcon, RelatedTopics, WebSearch in same file?

Reply:
@S2606 ^

Reply:
@chiragw15 I am working on it. Sorry for delay as i had exams

Reply:
@chiragw15 should i keep the file name as relatedtopic or change it??

Reply:
Keep file name anything but keep class names same.

Reply:
@chiragw15 The thing is on adding the other class it is showing an error

Reply:
@S2606 take help from these https://android.jlelse.eu/keddit-part-6-api-retrofit-kotlin-d309074af0  and
https://github.com/juanchosaravia/KedditBySteps/commit/8295dfa18896a3ac18757e4c026fa1e7d5ec1600#diff-f419ad5f6c76e2617a08405003a66c76